### PR TITLE
Update vscode.md to instruct Windows users to install GNU Make + inst…

### PR DESCRIPTION
**Pull request by janjongboom**
…ructions to build with CLI

Fix for https://github.com/ARMmbed/Handbook/issues/409

@iriark01 @AnotherButler

### CODE REVIEW

File: docs/tutorials/debug/vs_code.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 This document explains how to build and debug Arm Mbed OS applications using Visual Studio Code. Before starting, first [configure your local debug toolchain](/docs/v5.6/tools/setting-up-a-local-debug-toolchain.html).
 
+If you're using Windows, also install [GNU Make](https://www.gnu.org/software/make/).
```
[the remark this answers is not in the document]
Comment (loverdeg-ep) [reply docs/tutorials/debug/vs_code.md]: Not completely. Most linux distributions I believe have it installed by default.

[the remark this answers is not in the document]
Comment (theotherjimmy) [reply docs/tutorials/debug/vs_code.md]: Ubuntu does, but base Archlinux does not. Either way, it's worth mentioning.

[the remark this answers is not in the document]
Comment (theotherjimmy) [reply docs/tutorials/debug/vs_code.md]: Oh, and Mac does not come with make.

**Comment by janjongboom** [comment]
@theotherjimmy @AnotherButler Fixed, squashed and pushed.

**Pull request merged**